### DISCUSSION TITLE
Unable to find powershell core installed from windows11 store

### DISCUSSION THREAD

**Discussion by rise-worlds** [Eugeny/tabby#5175] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/464270/146290638-e809cb96-80ef-4ce8-8486-3c61f0c1fdc7.png)
![image](https://user-images.githubusercontent.com/464270/146290687-f2c38ab5-8f29-4ce5-8504-64ffa924774e.png)

**Comment by Eugeny** [comment, marked as answer]
Tabby is checking `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\pwsh.exe` - is it set?

**Reply by rise-worlds** [reply]
This path does not exist, and versions installed from the store will not be written here.

**Reply by Rakambda** [reply]
Microsoft Terminal detected `C:\\Users\\user\\AppData\\Local\\Microsoft\\WindowsApps\\Microsoft.PowerShell_8wekyb3d8bbwe\\pwsh.exe` for pwsh from the store.

**Reply by rise-worlds** [reply]
> Microsoft Terminal detected `C:\\Users\\user\\AppData\\Local\\Microsoft\\WindowsApps\\Microsoft.PowerShell_8wekyb3d8bbwe\\pwsh.exe` for pwsh from the store.

@Eugeny Can you check this path on load?

**Comment by lawweiliang** [comment]
@rise-worlds   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
